test/linter/test-browsers-data.js

```diff
@@ -0,0 +1,57 @@
+'use strict';
+const path = require('path');
+const chalk = require('chalk');
+const { Logger } = require('./utils.js');
+
+/**
+ * @typedef {import('../../types').Identifier} Identifier
+ * @typedef {import('../utils').Logger} Logger
+ */
+
+/**
+ * @param {Identifier} data
+ * @param {Logger} logger
+ * @returns {boolean}
+ */
+function processData(data, logger) {
+  const browser = Object.keys(data.browsers)[0];
+  const releases = data.browsers[browser].releases;
+
+  let releaseByStatus = {
```

Comment:
Is this unused?

Reply:
Looks like that was left over from a former revision and is no longer used -- removed!